functionary/ui/templates/core/task_detail.html

```diff
@@ -1,6 +1,18 @@
 {% extends "base.html" %}
 {% block content %}
     <div id="task_detail"
+        <div class="columns is-centered">
```

Comment:
I'm not sure what these changes are about.  I think you can just remove them.

Reply:
Fixed.